src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Pressable, StyleSheet, View, ViewStyle } from 'react-native';
+import { Menu as BaseMenu } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+import type { IconName } from '../icons/IconProps';
+
+type MenuOption = {
+    title: string;
+    onPress: () => void;
+};
+
+interface Props {
+    visible: boolean;
+    onPress: () => void;
+    onDismiss: () => void;
+    options: MenuOption[];
+    icon: IconName;
+    iconSize?: number;
+    iconColor?: string;
+    style?: ViewStyle;
+    menuStyle?: ViewStyle;
+    optionStyle?: ViewStyle;
+}
+
+export const Menu = ({
+    visible,
+    onPress,
+    onDismiss,
+    options,
+    icon,
+    style,
+    iconSize = 24,
+    iconColor,
+    menuStyle,
+    optionStyle,
+}: Props) => {
+    const theme = useTheme();
+    const styles = StyleSheet.create({
+        container: {
+            justifyContent: 'center',
+            ...style,
+        },
+        menu: {
+            paddingVertical: 24,
```

Comment:
don't we have padding in the theme ?

should be relying on theme dimension if possible